cg/meta/workflow/mutant/metrics_parser/models.py

```diff
@@ -0,0 +1,33 @@
+from typing import Annotated
+from pydantic import BaseModel, BeforeValidator, ValidationError
+
+
+def empty_str_to_none(v: str) -> str | None:
```

Comment:
```suggestion
def empty_str_to_none(value: str) -> str | None:
```
use descriptive parameter names